### PR TITLE
Bug 1877743: Update openshift/machine-api-operator and pull in k8s 1.19

### PR DESCRIPTION
**Pull request by zaneb**
Ensure we are vendoring the latest MAO.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,16 +3,30 @@ module github.com/openshift/cluster-api-provider-baremetal
...
 
 require (
+	github.com/fatih/color v1.9.0 // indirect
+	github.com/gobuffalo/flect v0.2.1 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/json-iterator/go v1.1.10 // indirect
+	github.com/mattn/go-colorable v0.1.7 // indirect
 	github.com/metal3-io/baremetal-operator v0.0.0
 	github.com/onsi/gomega v1.9.0
 	github.com/openshift/machine-api-operator v0.2.1-0.20200721125631-d234cceb5de1
```
Comment (michaelgugino) [inline go.mod:13]: This did not appear to update machine-api-operator depedency, actually.

Comment (zaneb) [reply go.mod:13]: Excellent point :facepalm:

**Comment by dhellmann** [comment]
@zaneb it looks like this needs to be linked to a bug report

**Comment by zaneb** [comment]
This is likely to be superseded by #109.

**Comment by JoelSpeed** [comment]
This now includes the changes from #109 and covers the same purpose, lets use this PR to get the changes merged. You can use bug 1877743 for this

**Comment by zaneb** [comment]
/retitle Bug 1877743: Update openshift/machine-api-operator and pull in k8s 1.19

**Pull request merged**